pennylane/workflow/qnode.py

```diff
@@ -23,6 +23,7 @@
 from collections.abc import Callable, Sequence
 from typing import Any, Literal, Optional, Union, get_args
 
+import pytest

```

Comment:
Why are you using pytest 😅 . `warnings` is already imported.

And the rest of the code in this file needs to be changed accordingly as well.

Reply:
Sorry my brain is broken